core_v2/sources/domains.move

```diff
@@ -636,11 +634,11 @@ module aptos_names_v2::domains {
     }
 
     /// this is for domain owner to update subdomain expiration time
-    public fun renew_subdomain_as_domain_owner(
+    public fun set_subdomain_expiration_as_domain_owner(
         sign: &signer,
         subdomain_name: String,
         domain_name: String,
-        registration_duration_secs: u64,
+        expiration_time_sec: u64,
     ) acquires CollectionCapabilityV2, NameRecordV2{
         validate_subdomain_to_renew(sign, subdomain_name, domain_name);
```

Comment:
what's the better name here? now the time stamp is a absolute timestamp instead of a time span. this name was suggested by copilot :P I couldn't think of a better one.